packages/app/vite.config.ts

```diff
@@ -28,13 +29,13 @@ export default defineConfig({
           if (id.includes("@firebase")) {
             return "f";
           }
-          if (id.includes("@ethersproject")) {
+          if (id.includes("ethers")) {
```

Comment:
I think that now since you're importing only functions you use instead of whole ethers like before, maybe you don't need to split it in a separate chunk. But not 100% sure